src/components/ButtonPanel.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import Button from './Button';
+
+const ButtonPanel = () => (
+  <div className="button-panel">
+    <div className="group-1">
+      <Button name="AC" />
+      <Button name="+/-" />
+      <Button name="%" />
+      <Button name="÷" />
+    </div>
+    <div className="group-2">
+      <Button name="7" />
+      <Button name="8" />
+      <Button name="9" />
+      <Button name="x" />
+    </div>
+    <div className="group-3">
+      <Button name="4" />
+      <Button name="5" />
+      <Button name="6" />
+      <Button name="-" />
+    </div>
+    <div className="group-4">
+      <Button name="1" />
+      <Button name="2" />
+      <Button name="3" />
+      <Button name="+" />
+    </div>
+    <div className="group-5">
+      <Button name="0" />
+      <Button name="." />
+      <Button name="=" />
```

Comment:
- [ ] Rather than this method of displaying Buttons kindly group the buttons then map through it. It would be more concise and neater:
```
Group 1: AC, +/-, %, ÷
Group 2: 7, 8, 9, X
Group 3: 4, 5, 6, -
Group 4: 1, 2, 3, +
Group 5: 0, ., =
```

Reply:
I made the changes requested, which includes:-
- Wrapping Display and ButtonPanel in React.Fragment
- Refactor ButtonPanel component to use map